### PR TITLE
News - gc-feeds - Plugin stabilization

### PR DESCRIPTION
**Pull request by jmealing**
- removed provisional class
- updated EN/FR examples

### CODE REVIEW

**Comment by Garneauma** [comment]
@jmealing After discussing with the team, we came to a conclusion where we are questioning the utility of the `.gc-feeds` component since the only thing it changes is that the feed links are bolded (all other styles don't do anything).

If the intention for the feed component is to **always** have bolded links, we could simply overwrite WET-BOEW's styles for the GCWeb theme instead of creating a new component. According to design decision 14 (https://wet-boew.github.io/wet-boew-documentation/decision/14.html), this would only be a patch since the visual difference is arguably barely noticeable for the average Canada.ca user.

If the intention for the feed component is to **never** have bolded links, we could simply get rid of the component altogether.

We can discuss this during next week's office hours if you prefer.

**Comment by jmealing** [comment]
@Garneauma @delisma The intention is to make the news items links always bold, to carry the same visual weight as the other links throughout the templates that use this pattern.

If there's a different way to go about it, then by all means. I've looped in David E in case there's a technical challenge or reason I'm not understanding

**Comment by jmealing** [comment]
@Garneauma This one isn't working for me to be able to do anything anymore. There is no folder for /wet-boew and when I try and rebase to get it there's all kinds of conflicts for stuff I didn't touch @duboisp @delisma

**Comment by jmealing** [comment]
Screenshot 2024-01-11 at 3 19 07 PM
Screenshot 2024-01-11 at 3 19 14 PM
Screenshot 2024-01-11 at 3 19 24 PM

**Comment by Garneauma** [comment]
@jmealing Did this PR only have the change listed in the "Files changed" tab? (components/gc-feeds/_base.scss)

@duboisp Would you know how to fix this? Should @jmealing simply `git reset --hard upstream/master` and reapply changes?

**Comment by jmealing** [comment]
I updated 3 files:
```
gc-feeds-fr.html
gc-feeds-en.html
/gc-feeds/_base.scss
```

The conflicts which won't let me merge are all within the `package-lock.json` file

File: wet-boew/feeds/index.json-ld
Comment (Garneauma) [inline wet-boew/feeds/index.json-ld:37]:
```suggestion
				"title": "Feeds",
				"language": "en",
				"url": "https://wet-boew.github.io/gcweb-compiled-demos/wetboew-demos/feeds/feeds-en.html"
```

File: wet-boew/feeds/index.json-ld
Comment (Garneauma) [inline wet-boew/feeds/index.json-ld:42]:
```suggestion
				"title": "Fils de syndication",
				"language": "fr",
				"url": "https://wet-boew.github.io/gcweb-compiled-demos/wetboew-demos/feeds/feeds-fr.html"
```

File: wet-boew/feeds/index.json-ld
Comment (Garneauma) [inline wet-boew/feeds/index.json-ld:23]:
```suggestion
				"title": "Feeds",
```

File: wet-boew/feeds/index.json-ld
Comment (Garneauma) [inline wet-boew/feeds/index.json-ld:25]:
```suggestion
				"url": "https://wet-boew.github.io/wet-boew/docs/ref/feeds/feeds-en.html"
```

File: wet-boew/feeds/index.json-ld
Comment (Garneauma) [inline wet-boew/feeds/index.json-ld:28]:
```suggestion
				"title": "Fils de syndication",
```

**Review by Garneauma — CHANGES_REQUESTED** [review]
This PR is very close to being merged. Please apply changes to the JSON-LD, then I think it'll be good!

Thank you @jmealing !

**Comment by jmealing** [comment]
@Garneauma This one can be merged when it is ready. We don't have guidance that directly links to or from this component

File: wet-boew/feeds/index.json-ld
Comment (Garneauma) [inline wet-boew/feeds/index.json-ld:14]:
```suggestion
		"en": "Aggregates and displays entries from one or more Web feeds.
--


",
```

File: wet-boew/feeds/index.json-ld
Comment (Garneauma) [inline wet-boew/feeds/index.json-ld:15]:
```suggestion
		"fr": "Affiche les articles d’un ou plus fils de syndication."
```

**Review by Garneauma — CHANGES_REQUESTED** [review]
@jmealing Can you accept those change requests? Sorry I hadn't noticed the description yesterday. Once it is done, I will approve this PR.

Thank you for your contribution!

**Review by Garneauma — APPROVED** [review]

**Comment by duboisp** [comment]
Pre-approved upon final/quick review by @duboisp

**Review by duboisp — APPROVED** [review]
Like the `h1.thickline` we need to note that the provisional feature `gc-feed` which as been conjointly used with the `wb-feed` plugin is now obsolete because it is the new default. That is not a blocker to merge this PR.

**Pull request merged**
